### DISCUSSION TITLE
Parameterübergabe an das shell-script für Benachrichtigungen

### DISCUSSION THREAD

**Discussion by bollermann** [camueller/SmartApplianceEnabler#513] [Installation und Betrieb]
Hallo zusammen,
beim Einrichten eines Benachrichtigungs-Scripts bin ich in die folgende Falle getappt: 
Mein DeviceName lautete "Wärmepumpe Pv+". Offensichtlich führte das blank im DeviceName zu Problemen bei der Parameterübergabe an das shell-script. Es wurde kein Benachrichtigungs-Text ausgegeben und der DeviceName nur bis zum blank. Nachdem das Leerzeichen getilgt wurde, funktioniert das script wie erwartet. Das "+" wird allerdings durch ein Leerzeichen ersetzt.

**Comment by peakMeissner** [comment]
Das Leerzeichen hat mir auch Probleme bereitet. Danke für den Beitrag, dass hat mir jedenfalls geholfen meinen Problem zu beheben.

**Comment by camueller** [comment]
Eigentlich sollte sich das lösen lassen, sodass Leeerzeichen möglich sind. Ich schaue mir das mal an.

**Comment by camueller** [comment]
Welches Messenger-Script verwendet ihr?

**Reply by bollermann** [reply]
ich verwende das Signal-Script.

**Reply by peakMeissner** [reply]
Ich benutze ebenfalls das Signal-Skript. Mein Gedanke wäre zu prüfen ob man die CURL-Parameter via --data-urlencode übergeben kann. Viele Grüße FCD

**Reply by camueller** [reply]
Ja, das wäre vermutlich die Lösung. Du kannst es ja ausprobieren und mir dann mitteilen oder einen Pull-Request schicken. Das Signal-Script wurde von einem SAE-User erstellt. Da ich Signal nicht nutze, habe ich es auch nicht getestet.
